### PR TITLE
FFA-41 implement fab button to search films

### PR DESCRIPTION
**Pull request by Coshiloco**
-Replace FloatingActionButton icon and disablle logOut function

### CODE REVIEW

File: app/src/main/java/com/digitalsolution/familyfilmapp/navigation/AppNavigation.kt [hunk 1]
```diff
@@ -35,7 +42,21 @@ fun AppNavigation() {
...
+                ) {
+                    Icon(
+                        imageVector = Icons.Default.Search,
+                        contentDescription = null,
+                        modifier = Modifier.padding(10.dp)
+                    )
+                }
+            }
+        },
+        floatingActionButtonPosition = FabPosition.End
```
Comment (rndevelo) [inline app/src/main/java/com/digitalsolution/familyfilmapp/navigation/AppNavigation.kt:59]: This is unnecessary, this parameter is by default.

**Review by rndevelo — CHANGES_REQUESTED** [review]
Look, only one thing

File: app/src/main/java/com/digitalsolution/familyfilmapp/navigation/AppNavigation.kt (hunk 1 above)
Comment (Coshiloco) [reply app/src/main/java/com/digitalsolution/familyfilmapp/navigation/AppNavigation.kt:59]: Without this parameter not ajust to the end correctly i tested

**Review by rndevelo — APPROVED** [review]

File: app/src/main/java/com/digitalsolution/familyfilmapp/repositories/FakeRepository.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        }
+
+        return films
+    }
+
+
+}
+
+
+interface FakeRepository {
```
Comment (hgarciaalberto) [inline app/src/main/java/com/digitalsolution/familyfilmapp/repositories/FakeRepository.kt:27]: @Coshiloco Can you change the name of this interface, please? Then I'll approve it right away!

**Review by hgarciaalberto — CHANGES_REQUESTED** [review]
Just a small change!

**Pull request merged**